docker/Dockerfile.base

```diff
@@ -3,14 +3,24 @@ ARG base_immage
 
 FROM $build_image AS build-image
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
"noninteractive – You use this mode when you need zero interaction while installing or upgrading the system via apt. It accepts the default answer for all questions. It might mail an error message to the root user, but that's it all. Otherwise, it is totally silent and humble, a perfect frontend for automatic installs."